python/basix/numba_helpers.py

```diff
@@ -0,0 +1,56 @@
+try:
+    import numba
+except ImportError:
+    raise RuntimeError("You must have numba installed to use the numba helper functions.")
+
+
+@numba.njit
+def apply_dof_transformation(entity_transformations, cell_type, entity_dofs,
+                             data, block_size, cell_info):
+    if cell_type == "interval":
```

Comment:
String comparison isn't so elegant - maybe return different functions depending on the cell type which can then be used without the internal check.